src/arduino-timer.h

```diff
@@ -96,6 +96,16 @@ class Timer {
         task = (Task)NULL;
     }
 
+    /* Cancel ALL timer tasks -- useful for testing */
```

Comment:
I had to reuse the same timer for multiple tests.  

I added a Timer<>::cancelAll() method so I could delete tasks where the task ID was forgotten.  I hate adding a new public interface for this, but I couldn't think of a workaround.

(I wanted to make a local test timer (a local variable in the test function), but Arduinos don't have enough stack space for that AND the test infrastructure, etc.)

If this change in public interface is unacceptable, reject the pull request.